client/src/app/tabs/form/__tests__/FormEditorSpec.js

```diff
@@ -709,8 +725,15 @@ describe('<FormEditor>', function() {
 
       form._emit('commandStack.changed');
 
+      var calls = onActionSpy.getCalls();
+
+      calls = calls.filter(function(call) {
+        return call.args[0] === 'lint-tab';
+      });
+
       // then
-      expect(onActionSpy).to.have.been.calledOnce;
+      expect(calls.length).to.be.eql(1);
+      expect(onActionSpy).to.have.been.calledWith('lint-tab');

```

Comment:
The second expect statement can never fail on runtime. Either there is no call with `lint-tab`, which makes the test fail on the first statement, or it passes for both of them.